### PR TITLE
Fix Screenshot Tests

### PR DESCRIPTION
**Pull request by alperozturk96**
- [x] Tests written, or not not needed

### CODE REVIEW

File: app/src/main/java/com/owncloud/android/ui/dialog/SslUntrustedCertDialog.java [hunk 1]
```diff
@@ -76,9 +76,7 @@ public static SslUntrustedCertDialog newInstanceForEmptySslError(SslError error,
         if (error == null) {
             throw new IllegalArgumentException("Trying to create instance with parameter error == null");
         }
-        if (handler == null) {
```
Comment (alperozturk96) [inline app/src/main/java/com/owncloud/android/ui/dialog/SslUntrustedCertDialog.java:79]: null control already exists for usage of handler. For screenshot tests no need to pass SslErrorHandler. Mockito throws error for it.

File: app/src/androidTest/java/com/owncloud/android/ui/trashbin/TrashbinActivityIT.kt [hunk 2]
```diff
@@ -60,7 +61,7 @@ class TrashbinActivityIT : AbstractIT() {
             sut.trashbinPresenter = TrashbinPresenter(trashbinRepository, sut)
             sut.runOnUiThread { sut.loadFolder() }
             shortSleep()
-            screenshot(sut)
+            screenshot(sut, UUID.randomUUID().toString())
```
Comment (tobiasKaminsky) [inline app/src/androidTest/java/com/owncloud/android/ui/trashbin/TrashbinActivityIT.kt:64]: I fear a random name will not work, as then the recorded screenshot has always a different name than the original one.
What about giving them a fixed name: "TrashbinActivityIT_normalUser"?

This is what usually is done automatically, but I assume due to usage of scenario this does not work.

**Comment by AndyScherzinger** [comment]
❗ REBASED ❗

**Comment by alperozturk96** [comment]
Out of date due to conflicts

**Pull request closed without merging**